### PR TITLE
bitcoincoid: Bugfix cancelOrder

### PR DESCRIPTION
**Pull request by Cengkaruk**
```
> node examples/js/cli.js bitcoincoid cancelOrder "4204708" "XRP/IDR" '{"side": "buy"}'
bitcoincoid.cancelOrder (4204708, XRP/IDR, [object Object])
{ success:    1,
   return: { order_id:    4204708,
                 type:   "buy",
                 pair:   "xrp_idr",
              balance: {          idr: "2320857",
                                  btc: "0.00036748",
                                  bch: "0.00000000",
                                  btg: "0.00000000",
                                  bts: "0.00000000",
                                  drk: "0.00000000",
                                 doge: "0.00000000",
                                  eth: "0.00000000",
                                  etc: "0.00000000",
                                ignis: "0.00000000",
                                  ltc: "0.00000000",
                                  nxt: "0.00000000",
                                  nem: "0.00000000",
                                  str: "0.00000000",
                                  ten: "0.00000000",
                                waves: "0.00000000",
                                  xrp: "0.00000000",
                                  xzc: "0.00000000",
                           frozen_idr: "0",
                           frozen_btc: "0.00000000",
                           frozen_bch: "0.00000000",
                           frozen_btg: "0.00000000",
                           frozen_bts: "0.00000000",
                           frozen_drk: "0.00000000",
                          frozen_doge: "0.00000000",
                           frozen_eth: "0.00000000",
                           frozen_etc: "0.00000000",
                         frozen_ignis: "0.00000000",
                           frozen_ltc: "0.00000000",
                           frozen_nxt: "0.00000000",
                           frozen_nem: "0.00000000",
                           frozen_str: "0.00000000",
                           frozen_ten: "0.00000000",
                         frozen_waves: "0.00000000",
                           frozen_xrp: "0.00000000",
                           frozen_xzc: "0.00000000"  } } }
```

### CODE REVIEW

**Comment by kroitor** [comment]
https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#derived-exchange-classes

> - all associative keys must be single-quoted strings everywhere, array['good'], array.bad

Otherwise it won't work in Python... Also we have to check if the key is there before dereferencing it, so you can't write:

```
if (params['foobar'])
```

... but you can write:

```
if ('foobar' in params)
```

And you can't write:

```
if (params['abc'] === 1)
```

... but you can write:

```
if ('abc' in params)
    if (params['abc'] === 1)
```

In other words, we're using a common subset of all languages – the most restricted subset of it.

**Comment by Cengkaruk** [comment]
Oke, my bad. I will rewrite :)

File: js/bitcoincoid.js [hunk 1]
```diff
@@ -310,9 +310,17 @@ module.exports = class bitcoincoid extends Exchange {
...
+            throw new ExchangeError (this.id + ' cancelOrder requires a symbol argument');
+        if ('side' in params)
+            if (!params['side'])
+                throw new ExchangeError (this.id + ' cancelOrder requires a side as params argument');
         await this.loadMarkets ();
+        let market = this.market (symbol);
         return await this.privatePostCancelOrder (this.extend ({
             'order_id': id,
+            'pair': market['id'],
+            'type': params['side']
```
Comment (kroitor) [inline js/bitcoincoid.js]: One single comma missing at the end of this line to satisfy the syntax checkers.

Comment (Cengkaruk) [reply js/bitcoincoid.js]: Next PR 💃

**Pull request merged**